src/shared/lib/MutationTableDownloadDataFetcher.ts

```diff
@@ -86,6 +94,17 @@ export class MutationTableDownloadDataFetcher implements ILazyMobXTableApplicati
         }
     }
 
+    private async fetchAllGenomeNexusFunctionalImpactData()
```

Comment:
@leexgh per your slack question, i think actually that the check for mutationData should be handled outside of this function, for example by the remoteData instance that consumes it.  this function should assume that there is data passed to it (mutationData should be passed as argument).  So ...
1. pass mutationData as arg
2. no check for mutatationData
3. should not be tagged async
4. perhaps this function is not necessary at all since all it will do is return invocation